Makefile

```diff
@@ -161,34 +161,9 @@ opentok:
 	fi
 
 fabric:
-	@if [ ! -d Frameworks/Fabric ]; then \
-		echo "Downloading Fabric v$(FABRIC_SDK_VERSION)"; \
-		mkdir -p Frameworks/Fabric; \
-		curl -N -L -o fabric.zip https://s3.amazonaws.com/kits-crashlytics-com/ios/com.twitter.crashlytics.ios/$(FABRIC_SDK_VERSION)/com.crashlytics.ios-manual.zip; \
-		unzip fabric.zip -d Frameworks/Fabric || true; \
-		rm fabric.zip; \
-	fi
-	@if [ -e Frameworks/Fabric/Fabric.framework ]; then \
-		echo "Fabric v$(FABRIC_SDK_VERSION) downloaded"; \
-	else \
-		echo "Failed to download Fabric SDK"; \
-		rm -rf Frameworks/Fabric; \
-	fi
+	bin/download_framework.sh Fabric $(FABRIC_SDK_VERSION) https://s3.amazonaws.com/kits-crashlytics-com/ios/com.twitter.crashlytics.ios/INSERT_SDK_VERSION/com.crashlytics.ios-manual.zip; \
```

Comment:
Is it possible to store the download url in an environment variable define at the top of this file? 🤔

Reply:
> One thing I noticed is that the Fabric cache on Circle seems to no longer be working. ie. although the `Frameworks/Fabric` path is restored, the script still re-downloads Fabric.
> 
> It might also speed up our builds if we start caching the `Frameworks/Stripe` path, I must have missed adding this when we upgraded the Stripe SDK 😓

Cool will add that and look at moving the URL to an env var at the top 👍